### PR TITLE
Pause Menu

### PR DESCRIPTION
**Pull request by mlytvynchuk**
Added a pause menu.

### CODE REVIEW

**Comment by mlytvynchuk** [comment]
Please review previous PRs before this one. (Based on navigation_updated)

File: app/src/main/java/com/google/droidjump/GameFragment.java [hunk 1]
```diff
@@ -48,19 +54,51 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         View rootView = inflater.inflate(R.layout.game_screen, container, /* attachToRoot= */ false);
         ((LinearLayout) rootView.findViewById(R.id.game_layout)).addView(gameView);
-        NavigationHelper.addOnBackPressedEventListener(activity, new StartFragment());
+        menuButton = rootView.findViewById(R.id.menu_button);
+        pauseLayout = rootView.findViewById(R.id.pause_layout);
+        menuButton.setOnClickListener(ignored -> {
```
Comment (saimonsaret) [inline app/src/main/java/com/google/droidjump/GameFragment.java:59]: nit: you can probably remove the brackets here and below:
```
menuButton.setOnClickListener(ignored -> onPause());
```

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -32,4 +32,6 @@
         The purpose of the game is to avoid obstacles, pass levels, earn achievements and have fun!
         \nTo make droid jump, tap on the left side of the screen.
         \nTo make droid crouch, tap and hold on the right side of the screen</string>
+    <string name="restart_text">restart</string>
```
Comment (saimonsaret) [inline app/src/main/res/values/strings.xml:35]: Should it be "Restart" and "Main menu"?

**Review by saimonsaret — DISMISSED** [review]

File: app/src/main/java/com/google/droidjump/GameFragment.java (hunk 1 above)
Comment (mlytvynchuk) [reply app/src/main/java/com/google/droidjump/GameFragment.java:59]: Fixed.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (mlytvynchuk) [reply app/src/main/res/values/strings.xml:35]: No difference because all buttons are uppercase styled but better to be titled. Fixed.

**Review by saimonsaret — APPROVED** [review]

**Pull request merged**